osu.Game/Overlays/News/NewsArticleCover.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+
+namespace osu.Game.Overlays.News
+{
+    public class NewsArticleCover : CompositeDrawable
```

Comment:
shouldn't this class handle showing the title/author/date?

![image](https://user-images.githubusercontent.com/191335/70392687-f6814300-1a25-11ea-9453-74008d2f0fc9.png)

Reply:
This class was intended to only provide async background image loading for a news article card (image shown above), whose implementation is to come soon (as there is no API struct avalaible for news articles to base the component on, unless it is fine to use a dummy struct to develop the component?).